lib/cli.js

```diff
@@ -118,6 +118,10 @@ exports.run = () => {
 		throw new Error(colors.error(figures.cross) + ' Watch mode is not available in CI, as it prevents AVA from terminating.');
 	}
 
+	if (hasFlag('--concurrency') || hasFlag('-c')) {
```

Comment:
No, that wouldn't work if the user had `concurrency` defined in package.json.

The correct check would be:

```js
if (cli.flags.concurrency === '') {
```

As it would be an empty string if the value is not defined, as we already define `concurrency` to be a string in the `meow` config.